lib/target/m100/def/romcalls.def

```diff
@@ -0,0 +1,32 @@
+; M100 firmware macros and defs
+; Applicable to American M100/M102 ONLY (probably, I've never seen any European M100)
+; Nov 2020
+;
+; M100 system ROM aka firmware can be called in atleast a couple of ways
+; If we're calling it from OPTROM made with Stephen Adolph's template, we do
+; RST 6; DEFW ROM_ADDR
+; And if we're calling from normal RAM, then we simply do CALL ROM_ADDR
+
+; With ROMCALL macro, all ROM calls (except RSTs) can be abstracted into
+; ROMCALL; DEFW ROM_ADDR
+; ...and be conditionally compiled depending on the subtype we use.
+
+#define ROMCALL defb ROMCALL_OP
+
+IF __OPTROM__
+ROMCALL_OP EQU $F7 ; rst 6 if calling from OPTROM
```

Comment:
I think, that if you compile the library with EXTERN ROMCALL_OP and then defc ROMCALL_OP (and PUBLIC it) in the crt file, then we'll only need a single library, z80asm should resolve the value at link time.